fix Source0 and URL

fbrosson opened the pull request:
fix Source0 and URL as the project has moved from berlios.de to sourceforge.net

fbrosson committed:
fix Source0 and URL

fix Source0 and URL as the project has moved from berlios.de to sourceforge.net

fbrosson commented:
Sorry for this duplicate PR. The previous got automaticaly closed when I tried to fix my copy of master. (That won'r happen again.)

robxu9 commented on .abf.yml:
```diff
@@ -1,2 +1,2 @@
 sources:
-  cdrtools-3.01.tar.bz2: 62766aa9bfeec91b3c78da7c59b5c96bfbec84d9
+  cdrtools-3.02a01.tar.bz2: a7fccb3b16c7a4fc49d1926d72f20ebba290029a
```
This should be updated when you abf put to the filestore. If you changed this manually, this will not work.

fbrosson replied:
Indeed, I changed this manually. I though the 2 files (cdrtools.spec and .abf.yml) had to be changed with a single commit. And I also did not know about the filestore and "abf put". I haven't read the docs yet and cannot promisse I'll have time to do so in the near future. So please feel free to do whatever you wish with this PR. Thanks in advance if you can do something with my diff. Cheers

fbrosson commented:
Hi @robxu9 and @tpgxyz! I've updated `cdrtools.spec` to 3.02a02 and reverted my (wrong) edits in `.abf.yml` in case someone would like to merge my commit and launch the `abf put` command. Thanks!

fbrosson commented:
Hi @robxu9 and @tpgxyz ,
mkisofs supports DVD-Audio since cdrtools-3.02a04.
